src/main/java/Terry.java

```diff
@@ -0,0 +1,139 @@
+import task.*;
+
+import java.util.Scanner;
+import ui.Ui;
+import storage.Storage;
+
+public class Terry {
+
+    private Storage storage;
+    private Ui ui;
+
+    private static final int COMMAND_AND_DETAILS_LIMIT = 2;
+
+    public static void main(String[] args) throws TerryException {
+
+        Ui ui = new Ui();
+        Storage storage = new Storage();
+        Ui.welcomeMessage();
+
+        Scanner in = new Scanner(System.in);
+        String line;
+        UserInputList list = new UserInputList();
+        storage.loadTasksFromFile(list.getTasks());
+
+        while (true) {
+            line = in.nextLine().trim();
+            String[] words = line.split(" ", COMMAND_AND_DETAILS_LIMIT);
+            String command = words[0].toLowerCase();
+
+            if (process_command(command, list, words, line, ui, storage)) return;
+        }
+
+    }
+
+    private static boolean process_command(String command, UserInputList list, String[] words, String line, Ui ui, Storage storage) throws TerryException {
```

Comment:
The java coding standard we follow does not permit use usage of `snake_case` for naming functions. Use `camelCase` only.